plugins/hls-refactor-plugin/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -5,6 +5,14 @@
 {-# LANGUAGE GADTs                  #-}
 {-# LANGUAGE RankNTypes             #-}
 {-# LANGUAGE TypeFamilies           #-}
+{-# LANGUAGE LambdaCase             #-}
+{-# LANGUAGE TupleSections          #-}
+{-# LANGUAGE ViewPatterns           #-}
+{-# LANGUAGE ScopedTypeVariables    #-}
+{-# LANGUAGE RecordWildCards        #-}
+{-# LANGUAGE TypeApplications       #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
I presume that these were all listed in the `ghcide.cabal` descriptor? I would prefer to copy that approach for `hls-refactor` too, these long lists of common extensions are silly.